packages/block-library/src/quote/v2/index.js

```diff
@@ -30,37 +30,7 @@ const settings = {
 	transforms,
 	edit,
 	save,
+	deprecated,
 };
 
 export default settings;
-
-/**
- * This function updates the attributes for the quote v2.
- * This should be moved to block.json when v2 becomes the default.
- *
- * @param {Array}  blockSettings The settings of the block to be registered.
- * @param {string} blockName     The name of the block to be registered.
- * @return {Array} New settings.
- */
-function registerQuoteV2Attributes( blockSettings, blockName ) {
```

Comment:
Had to remove the filter because it was affecting the deprecation, preventing it from having the `value` attribute to migrate. The result of this change is that when v2 is enabled, the `value` attribute will still be present.

We need to remove the attribute from the `block.json` when v2 becomes the default ([see](https://github.com/WordPress/gutenberg/pull/25892#issuecomment-1076686005)).

Reply:
> We need to remove the attribute from the block.json when v2 becomes the default

I don't think we can remove it until the `useMigrateOnLoad` is removed.